README.md

```diff
@@ -1 +1,184 @@
-# HTML-CSS-JavaScript-capstone-project---Conference-page
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+  <h3><b>First Microverse Capstone Project</b></h3>
+
+</div>
+
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [📖 Video Description](#vid-desc)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Conference Page <a name="about-project"></a>
+
+**Conference page** is the first Microverse Capstone at the end of Module 1
+
+## Video Description <a name="vid-desc">
+[Loom video describing project](#)
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+HTML, CSS and JavaScript
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://nodejs.org/">Node.js</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="#">Live Server</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **Navigation Bar**
+- **Mobile Menu**
+- **Dynamic HTML for speakers section**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://flemton.github.io/capstone1/)
+
```

Comment:
- Please, ensure you've deployed the correct project as your deployment is showing another project different from your original project.

Reply:
@zieeco Thanks for the suggestion. Please check the commit message. It is fixed in README.md file.